JavaErickHernandez/Day3/ArrayLists.java

```diff
@@ -0,0 +1,93 @@
+package bootcamp;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.*;;
+
+public class ArrayLists {
+	static ArrayList<number> numbers = new ArrayList<number>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, userSet, etc...

Reply:
**Use List<> nameList = new ArrayList<>();** Almost always List is preferred over ArrayList because, for instance, List can be translated into a LinkedList without affecting the rest of the codebase.

If one used ArrayList instead of List, it's hard to change the ArrayList implementation into a LinkedList one because ArrayList specific methods have been used in the codebase that would also require restructuring: https://docs.oracle.com/javase/tutorial/collections/implementations/list.html. **Correct everyone who has the same problem**

Reply:
Into Java 8 is not necesary (List<number> numberList = new ArrayList**<number>**();) you can put only (List<number> numberList = new ArrayList<>();)

Reply:
Thanks for the advice